docs/interfaces/Emittable.md

```
@@ -0,0 +1,144 @@
+##### Parameters
+
+| Parameter | Type                                                                          | Description                                                       |
+| --------- | ----------------------------------------------------------------------------- | ----------------------------------------------------------------- |
+| `event`   | `"*"`                                                                         | The wildcard event type "\*"                                      |
+| `handler` | [`WildcardEventHandler`](../type-aliases/WildcardEventHandler.md)\<`Events`\> | A [WildcardEventHandler](../type-aliases/WildcardEventHandler.md) |
+
+##### Returns
+
+`void`
+
+#### Call Signature
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)